spec/exp_descriptor_en_spec.rb

```diff
@@ -42,6 +42,10 @@ def desc(expression, opts = {})
         expect(desc('*/5 * * * *')).to eq('Every 5 minutes')
       end
 
+      it 'every 5 minutes at Midnight' do
+        expect(desc('*/5 0 * * *')).to eq('Every 5 minutes, at 12:00 AM')
```

Comment:
Ideally it should say `Every 5 minutes, between 12:00 AM and 12:59 AM`, like in the attached screenshot example.

`Every 5 minutes, at 12:00 AM` sounds a bit confusing/unclear.

Reply:
I agree. And there are some other cases that are unchanged by this PR which are also confusing, such as the one on line 367. Since "at 1:00 AM" is not really what's happening.

Perhaps we can fix all of those at once: When a single hour is given use the range of minutes that compose that full hour.

I will look into it.

Reply:
yeah, fixing the  wording from `at X:00 AM` to `between HH:00 and HH:59 AM` for the `Hour` is worth a separate issue.